### ISSUE TITLE
JC logo in header missing alt text

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#195]
I have been working my way through the site doing some accessibility checks and the WAVE tool is saying that the main JC logo in the header is missing alt text. Thanks!

**Comment by joelworsham** [comment]
I thought I had this fixed, then I looked into the code and realized the *mobile* logo did not have alt text.

All set!

**Comment by banksashleym** [comment]
Thanks so much!
